open_spiel/python/algorithms/best_response.py

```diff
@@ -152,8 +198,12 @@ def best_response_action(self, infostate):
     infoset = self.infosets[infostate]
     # Get actions from the first (state, cf_prob) pair in the infoset list.
     # Return the best action by counterfactual-reach-weighted state-value.
+    def legal_actions_wrapper(state):
```

Comment:
There's no need for this indirection. We just need a list of legal actions to pass to `max`.
I think state.legal_actions(self.player_id) would work in all cases, but if not we can construct the list here.

Reply:
I seems to me that in the case of sequential games, best_response_action can be called on decision nodes of other players. However, in the case of simultaneous games, we would like to do the max over actions of the player _player_id.

Reply:
Sorry, I understand this remark now. It has been changed.